tests/system/Database/Live/UpdateTest.php

```diff
@@ -278,7 +278,7 @@ public function testUpdateBatchConstraintsRawSqlandAlias()
             [
                 'id'      => 2,
                 'name'    => 'Ahmadinejad Changes',
-                'country' => 'Iraq', // same length but different
+                'country' => 'Uruguay',
```

Comment:
Why did you remove the existing test case?

Reply:
I'm not where I removed any tests. I looked back through the commits and didn't see any. Sometimes I'll rename them if I add to them. In general though I try to build on existing tests where appropriate to try and not create redundant tests. I try and get the maximum code coverage for each test.

Reply:
I searched `// same length but different` in UpdateTest.php, but not found.
You had a test case `Iraq` for *same length but different*, but it was removed
because `Uruguay` is not the same length.
Is it okay?

Reply:
I changed this test around a few times.. I was having trouble useing RawSql with the % character. Ultimately I fixed the issue and changed the test to do what I was originally wanting to do.

Reply:
I changed to Uruguay and used LIKE 'U%'